dynawo/sources/Modeler/DataInterface/DYNGeneratorInterface.h

```diff
@@ -94,6 +94,12 @@ class GeneratorInterface : public ComponentInterface, public ReactiveCurvePoints
    */
   virtual double getQMax() = 0;
 
+  /**
+   * @brief Getter for the maximum reactive power of the generator diagram
+   * @return The maximum reactive power of the generator diagram in Mvar (generator convention)
```

Comment:
I'd explicit the comment specifying that this retrieves the maximum value of diagram reactive capability curve, whatever injected P. To make the distinction with the already existing getQMax() method